presets/inference/vllm/tests/test_inference_api.py

```diff
@@ -0,0 +1,114 @@
+import sys
+import os
+import subprocess
+import time
+import socket
+from pathlib import Path
+
+import pytest
+import requests
+
+# Get the parent directory of the current file
+parent_dir = str(Path(__file__).resolve().parent.parent)
+# Add the parent directory to sys.path
+sys.path.append(parent_dir)
+
+TEST_MODEL = "facebook/opt-125m"
+CHAT_TEMPLATE = ("{{ bos_token }}{% for message in messages %}{% if (message['role'] == 'user') %}"
+    "{{'<|user|>' + '\n' + message['content'] + '<|end|>' + '\n' + '<|assistant|>' + '\n'}}"
+    "{% elif (message['role'] == 'assistant') %}{{message['content'] + '<|end|>' + '\n'}}{% endif %}{% endfor %}")
```

Comment:
so in future pr - controller side - I believe we will have to have a default chat template for each model

Reply:
for those model which doesn't have a default chat template. we should provide one.